src/coll/error.rs

```diff
@@ -320,17 +332,19 @@ impl BulkWriteException {
         // Parse out any write errors.
         let w_errs = if let Some(&Bson::Array(ref errors)) = result.get("writeErrors") {
             if errors.is_empty() {
-                return Err(Error::ResponseError(
-                    "Server indicates a write error, but none were found.".to_owned()));
+                return Err(Error::ResponseError("Server indicates a write error, but none were \
+                                                 found."
+                    .to_owned()));
```

Comment:
Yeah, that is quite odd...

From looking at it, I think that should even be changed to use `String::from` (which got stabilized since last summer). Maybe I should do a search for all of the `.to_owned` calls and see which ones are replaceable?